src/promptflow-tools/connections.json.example

```diff
@@ -19,12 +19,13 @@
   "custom_connection": {
     "type": "CustomConnection",
     "value": {
-      "key1": "hey",
-      "key2": "val2"
+      "endpoint_url": "custom-endpoint-url",
+      "model_family": "GPT2",
+      "endpoint_api_key": "custom-endpoint-api-key"
     },
     "module": "promptflow.connections",
     "secret_keys": [
-      "key1"
+      "endpoint_api_key"
```

Comment:
Maybe you need to add a new connection named like `gpt2_connection` instead of modifying previous one?